pkgs/tools/misc/fpp/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, bzip2, python27Full, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  version = "0.7.1";
+  name = "fpp-${version}";
+
+  src = fetchFromGitHub {
+    owner = "facebook";
+    repo = "PathPicker";
+    rev = "refs/tags/${version}";
+    sha256 = "1mfyr9k5s3l1sg3c9vlyiqg8n1wwppzb981az2xaxqyk95wwl1sa";
+    name = "fpp-${version}";
+  };
+
+  buildInputs = [ bzip2 makeWrapper ];
+
+  buildPhase = ''
+    mkdir -p "$out/bin"
+    mv "fpp" "src" "$out"
+    ln -s "$out/fpp" "$out/bin/fpp"
+  '';
+
+  installPhase = ''
```

Comment:
Also, I think `buildPhase` really belongs in the `installPhase` here; if the standard build phase is causing problems here, simply set `dontBuild = true` to disable it.